src/api/items.test.tsx

```diff
@@ -242,7 +246,7 @@ describe('items api functions', () => {
     beforeEach(() => {
       moveItemsToSystem = {
         // Prevent test interference if modifying the usage statuses or selected items
-        usageStatusId: '0',
+        usageStatusConfig: '0',
```

Comment:
```suggestion
        usageStatusConfig: '0',
```
What was this changed from `usageStatusId` to `usageStatusConfig` ? isn't it still just the id

Reply:
No it is either a single ID, or a list of usage statuses and the linked item ID, hence the name config. I thought it was cleaner to put them in a single parameter rather than split them when handling the request to the api